examples/experimental/onnx/mobilenet_v2/requirements.txt

```diff
@@ -0,0 +1,2 @@
+torch==1.12.1
+torchvision==0.13.1
```

Comment:
torchvision>=0.12, <0.14

Reply:
What about torch? should it be aligned with torchvsion?

Reply:
Done